File: app/src/main/java/com/fenger/fragmentdemo/Fragments/LifeFragment.java
package com.fenger.fragmentdemo.Fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Toast;

import androidx.fragment.app.Fragment;

import com.fenger.fragmentdemo.customview.CustomerKeyboard;
import com.fenger.fragmentdemo.customview.PasswordEditText;
import com.fenger.fragmentdemo.R;

/**
 * com.fenger.fragmentdemo.Fragments
 * Created by fenger
 * in 2020-01-06
 */
public class LifeFragment extends Fragment implements CustomerKeyboard.CustomerKeyboardClickListener, PasswordEditText.PasswordFullListener {

    private CustomerKeyboard mCustomerKeyboard;
    private PasswordEditText mPasswordEditText;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_life, container, false);
        mPasswordEditText = view.findViewById(R.id.password);
        mCustomerKeyboard = view.findViewById(R.id.keyboard);

        mCustomerKeyboard.setOnCustomerKeyboardClickListener(this);
        mPasswordEditText.setOnPasswordFullListener(this);
        mPasswordEditText.setEnabled(false);

        return view;
    }

    @Override
    public void click(String num) {
        mPasswordEditText.addPassword(num);
    }

    @Override
    public void delete() {
        mPasswordEditText.deleteLastPassword();
    }

    @Override
    public void PasswordFull() {
        Toast.makeText(getContext(), "密码填充完毕：" + mPasswordEditText.getText(), Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/fenger/fragmentdemo/fragments/LifeFragment.kt
package com.fenger.fragmentdemo.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Toast
import androidx.fragment.app.Fragment
import com.fenger.fragmentdemo.R
import com.fenger.fragmentdemo.customview.CustomerKeyboard
import com.fenger.fragmentdemo.customview.CustomerKeyboard.CustomerKeyboardClickListener
import com.fenger.fragmentdemo.customview.PasswordEditText
import com.fenger.fragmentdemo.customview.PasswordEditText.PasswordFullListener

/**
 * com.fenger.fragmentdemo.Fragments
 * Created by fenger
 * in 2020-01-06
 */
class LifeFragment : Fragment(), CustomerKeyboardClickListener, PasswordFullListener {

    private lateinit var mCustomerKeyboard: CustomerKeyboard
    private lateinit var mPasswordEditText: PasswordEditText

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_life, container, false)
        mPasswordEditText = view.findViewById(R.id.password)
        mCustomerKeyboard = view.findViewById(R.id.keyboard)
        mCustomerKeyboard.setOnCustomerKeyboardClickListener(this)
        mPasswordEditText.setOnPasswordFullListener(this)
        mPasswordEditText.isEnabled = false
        return view
    }

    override fun click(number: String) {
        mPasswordEditText.addPassword(number)
    }

    override fun delete() {
        mPasswordEditText.deleteLastPassword()
    }

    override fun PasswordFull() {
        Toast.makeText(context, "密码填充完毕：" + mPasswordEditText.text, Toast.LENGTH_SHORT).show()
    }
}
